### ISSUE TITLE
wrap-with-parens and same those does not work see Attach

### ISSUE THREAD

**Issue by romankrv** [Fuco1/smartparens#931]
## Expected behavior
wrap-with-parens 
wrap-with-brackets
wrap-with-parens
wrap-with-braces
wrap-with-single-quotes
wrap-with-double-quotes
wrap-with-underscores
wrap-with-back-quotes
These functions not works
Log message like this: 
  command-execute: Autoloading file 
  /Users/admin/Documents/PROJECTS/conf/.emacs.d/elpa/smartparens-20180912.1750/smartparens.elc 
  failed to define function wrap-with-underscores

## Steps to reproduce the problem
M-x wrap-with-pare
![screen shot 2018-09-29 at 14 41 15](https://user-images.githubusercontent.com/42733/46247368-f0033e80-c412-11e8-9529-99e7c58965ec.png)
ns RET
I see in Emacs *MESSAGE* buffer:

command-execute: Autoloading file /Users/admin/Documents/PROJECTS/conf/.emacs.d/elpa/smartparens-20180912.1750/smartparens.elc failed to define function wrap-with-double-quotes

## Backtraces if necessary (`M-x toggle-debug-on-error`)
Debugger entered--Lisp error: (error "Autoloading file /Users/admin/Documents/PROJECTS/conf/.emacs.d/elpa/smartparens-20180912.1750/smartparens.elc failed to define function wrap-with-underscores")
  autoload-do-load((autoload "smartparens" nil t nil) wrap-with-underscores)
  command-execute(wrap-with-underscores)

## Environment & version information

**In recent enough `smartparens` you can call `M-x sp-describe-system` to generate this report**.  Please fill manually what we could not detect automatically.  Edit the output as you see fit to protect your privacy.

- `smartparens` version: smartparens-20180912.1750
- Active major-mode: Emacs-Lisp
- Emacs version (`M-x emacs-version`): GNU Emacs 26.1
- Spacemacs/Evil/Other starterkit (specify which)/Vanilla: GNU
![screen shot 2018-09-29 at 14 41 15](https://user-images.githubusercontent.com/42733/46247089-f8f21100-c40e-11e8-93c2-0acdf9cefe11.png)

- OS: macosx 10.13.16

**Comment by Fuco1** [comment]
Yes, this fails because those functions do not exist.  Did you copy your config from someone else? Maybe they have them defined locally.

*ALL* smartparens functions always start with `sp-` prefix so that's a good hint.

**Comment by Fuco1** [comment]
Yes this looked familiar, here's a duplicate issue https://github.com/Fuco1/smartparens/issues/899

So you guys are definitely getting this copied from somewhere.  The previous OP did not provide the source though.
